src/core/installAddon.js

```diff
@@ -333,6 +334,29 @@ export class WithInstallHelpers extends React.Component {
     }
   }
 
+  getCurrentStatus() {
+    const {
+      _addonManager,
+      addon: { guid },
+      hasAddonManager,
+    } = this.props;
+
+    if (!hasAddonManager) {
+      log.info('No addon manager, cannot set add-on status');
```

Comment:
I am not sure we need this because other methods don't have it, since we'll use them only when we have mozAddonManager.